dart/lib/flat_buffers.dart

```diff
@@ -111,7 +113,8 @@ class Builder {
 
   /// The list of existing VTable(s).
   //final List<_VTable> _vTables = <_VTable>[];
-  final List<int> _vTables = <int>[];
+  final List<int> _vTables = List<int>.filled(16, 0, growable: true)
```

Comment:
If you don't mind, can you remove the commented out line above this one while you're here?

Also - why are we changing this here? Are there benchmarks that showing a starting size of 16 is better than whatever the core SDK decides to start with?

Reply:
Sure, removed the commented out code.

Similarly to https://github.com/google/flatbuffers/pull/6696#discussion_r655596207 - don't have plain benchmarks with and without this change at the moment, only measured as a whole as part of the larger change suite suggested by mraleph a few months back.